PlaceRouteHierFlow/router/A_star.cpp

```diff
@@ -193,14 +193,14 @@ bool A_star::found_near_node(int current_node, Grid &grid, std::vector<int> &can
     east_found = expand_node(grid.vertices_total[current_node].east, east_node, grid);
     west_found = expand_node(grid.vertices_total[current_node].west, west_node, grid);
 
-    //if(grid.vertices_total[current_node].via_active_up){
-    if(1){
+    if(grid.vertices_total[current_node].via_active_up){
+    //if(1){
```

Comment:
Is this line kept for debugging purposes?

Reply:
That part is redundant. Has been removed.